mockserver/public/v3/state

```diff
@@ -1 +0,0 @@
ass": 29.9, "coal": 0, "geothermal": 0, "wind": 1562.1000000000001, "hydro": 2.0}, "exchangeCo2Intensities": {"AUS-VIC": 14.483333570758761}, "countryCode": "AUS-SA", "exchange": {"AUS-VIC": -251.73802}, "storage": {"battery": 0.0}, "datetime": "2017-12-06T20:25:00+10:00", "source": "aremi.nationalmap.gov.au, pv-map.apvi.org.au", "maxProduction": 1290.481, "production": {"oil": 0.0, "nuclear": 0, "gas": 1290.481, "biomass": 0.0, "coal": 0, "geothermal": 0, "wind": 336.252166, "hydro": 0.0}, "co2intensity": 14.483333570758761, "totalProduction": 1626.733166}, "BR-S": {"countryCode": "BR-S", "exchange": {"AR": 0.0077993, "UY": 60.47333, "PY": 0.0, "BR-CS": 5394.366}, "storage": {"hydro": null}, "datetime": "2017-10-05T08:31:00-03:00", "source": "ons.org.br", "production": {"unknown": 993.835938, "hydro": 4034.39233, "solar": 0.0, "wind": 1447.15942, "nuclear": 0.0}, "co2intensity": 114.16226305232769}, "PY": {"exchange": {"BR-S": -0.0, "BR-CS": 0.0}}, "DE": {"exchangeCo2Intensities": {"SE": 107.23353790918422}, "exchange": {"SE": 71.0}}, "LT": {"exchangeCo2Intensities": {"SE": 107.23353790918422}, "exchange": {"SE": 732.0}}, "AR": {"countryCode": "AR", "exchange": {"BR-S": -0.0077993}, "storage": {"hydro": null}, "datetime": "2017-10-05T07:00:00.233451-03:00", "source": "portalweb.cammesa.com", "production": {"oil": 472.2, "nuclear": 737.0, "gas": 8046.7, "biomass": 0, "coal": 311.0, "geothermal": 0.0, "solar": null, "unknown": 11.100000000000193, "wind": null, "hydro": 3920.0}, "co2intensity": 337.7410457816576}, "GB": {"exchangeCo2Intensities": {"IE": null}, "exchange": {"IE": -173.0}}, "SE": {"exchangeCo2Intensities": {"DK": 107.23353790918422, "NO": 107.23353790918422, "DE": 107.23353790918422, "LT": 107.23353790918422, "FI": 107.23353790918422, "PL": 107.23353790918422}, "countryCode": "SE", "exchange": {"DK": 411.0, "NO": 194.0, "DE": -71.0, "LT": -732.0, "FI": -1983.0, "PL": -595.0}, "storage": {}, "datetime": "2017-12-14T09:43:00+00:00", "source": "driftsdata.stattnet.no", "maxProduction": 10912.0, "production": {"nuclear": 8626.0, "unknown": 2046.0, "wind": 1420.0, "hydro": 10912.0}, "co2intensity": 107.23353790918422, "totalProduction": 23004.0}, "BR-NE": {"exchange": {"BR-CS": -235.9166}}, "BR-N": {"exchange": {"BR-CS": -1702.00964}}, "FI": {"exchangeCo2Intensities": {"SE": 107.23353790918422}, "exchange": {"SE": 1983.0}}, "IE": {"exchangeCo2Intensities": {"GB": 491.87272191893015, "GB-NIR": 491.87272191893015}, "countryCode": "IE", "exchange": {"GB": 173.0, "GB-NIR": 83.8}, "storage": {"hydro": 0}, "datetime": "2017-10-09T09:30:00+00:00", "source": "entsoe.eu", "production": {"oil": 401.0, "nuclear": null, "gas": 1652.0, "biomass": 238.0, "coal": 456.0, "geothermal": null, "solar": null, "unknown": 1.0, "wind": 385.0, "hydro": 74.0}, "co2intensity": 491.87272191893015}, "PL": {"exchangeCo2Intensities": {"SE": 107.23353790918422}, "exchange": {"SE": 595.0}}, "GB-NIR": {"exchangeCo2Intensities": {"IE": null}, "exchange": {"IE": -83.8}}}}}
```

Comment:
it seems like this file was deleted?

Reply:
My bad, I thought this was only generated for the mockserver

Reply:
yeah but let's keep it for the next contributors to be able to test. Can you restore and replace by the file where CA has been processed?